NuRadioMC/SignalGen/ARZ/ARZ.py

```diff
@@ -809,69 +846,9 @@ def get_time_trace(self, shower_energy, theta, N, dt, shower_type, n_index, R,
         tstart = t0 + tcenter
         i0 = np.int(np.round(tstart / dt))
         trace2[i0:(i0 + len(trace))] = trace
-        
+
         trace2 *= self._library['meta']['R'] / R * rescaling_factor
-        
+
         if(output_mode == 'Xmax'):
             return trace2, Lmax
         return trace2
-
-
```

Comment:
Why did you remove the __main__ example? This was an easy way to test things, right?

Reply:
The test module and the files in the tests folder do the same, only better.
The main function was broken anyway, and the example was not really
useful...